### ISSUE TITLE
Improve contrast for better accessibility

### ISSUE THREAD

**Issue by jmbowman** [pytube/pytube#78]
A couple of parts of the site have text with poor enough contrast against the background to be flagged as an accessibility concern:

* The copyright statement in the footer (all pages)
* Disabled entries in the jquery-listnav plugin (on the events, speakers, and tags pages)

The styling of this content should be updated so as to comply with the [WCAG minimum contrast](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) guidelines.  (Note that there are some special considerations for disabled user interface components that may be relevant for the listnav.)

**Comment by jmbowman** [comment]
This was fixed with https://github.com/pytube/pytube/pull/81 .
